solutions/pytorch_tests/Makefile

```diff
@@ -0,0 +1,135 @@
+
+.PHONY: push
+push:
+	$(DOCKER_PUSH)
+
+appdir:
+	$(APPBUILDER) -i $(DOCKER_FULL_NAME):$(DOCKER_TAG) -v
+
+rootfs: appdir
+	$(MYST) mkext2 appdir rootfs
+
+.PHONY: run-test-autograd
+run-test-autograd: rootfs
+	$(RUN_PYTEST) /workspace/pytorch/test/test_autograd.py -k "not test_deep_reentrant and not test_no_grad_copy_sparse and not test_profiler_seq_nr and not test_record_function and not test_thread_shutdown" 2>&1 | tee $@.out
+	grep -q -E '400 passed.*68 skipped.*7 deselected.*1 xfailed' $@.out
+
+.PHONY: run-test-autograd-thread-shutdown
+run-test-autograd-thread-shutdown: rootfs
+	$(RUN_PYTEST) /workspace/pytorch/test/test_autograd.py -k "test_thread_shutdown"
+
+.PHONY: run-test-modules
+run-test-modules: rootfs
+	$(RUN_PYTEST) /workspace/pytorch/test/test_modules.py 2>&1 | tee $@.out
+	grep -q -E '76 passed.*28 skipped' $@.out
+
+.PHONY: run-test-nn
+run-test-nn: rootfs
+	$(RUN_PYTEST) /workspace/pytorch/test/test_nn.py -k "not test_EmbeddingBag_empty_per_sample_weights_and_offsets_cpu_int32_int32_float32 and not test_EmbeddingBag_empty_per_sample_weights_and_offsets_cpu_int32_int64_float32 and not test_EmbeddingBag_empty_per_sample_weights_and_offsets_cpu_int64_int32_float32 and not test_EmbeddingBag_empty_per_sample_weights_and_offsets_cpu_int64_int64_float32 and not test_EmbeddingBag_per_sample_weights_and_new_offsets_cpu_int32_int32_float32 and not test_EmbeddingBag_per_sample_weights_and_new_offsets_cpu_int32_int64_float32 and not test_EmbeddingBag_per_sample_weights_and_new_offsets_cpu_int64_int32_float32 and not test_EmbeddingBag_per_sample_weights_and_new_offsets_cpu_int64_int64_float32 and not test_EmbeddingBag_per_sample_weights_and_no_offsets_cpu_int64_float32 and not test_EmbeddingBag_per_sample_weights_and_no_offsets_cpu_int32_float32 and not test_EmbeddingBag_per_sample_weights_and_offsets_cpu_int32_int32_float32 and not test_EmbeddingBag_per_sample_weights_and_offsets_cpu_int32_int32_float32 and not test_EmbeddingBag_per_sample_weights_and_offsets_cpu_int32_int64_float32 and not test_embedding_bag_1D_padding_idx_cpu_float32 and not test_EmbeddingBag_per_sample_weights_and_offsets_cpu_int64_int32_float32 and not test_embedding_bag_device_cpu_int32_int32_float32 and not test_EmbeddingBag_per_sample_weights_and_offsets_cpu_int64_int64_float32 and not test_embedding_bag_device_cpu_int32_int64_float32 and not test_embedding_bag_device_cpu_int64_int32_float32 and not test_embedding_bag_device_cpu_int64_int64_float32 and not test_embedding_bag_non_contiguous_weight_cpu_int32_int32_float32 and not test_embedding_bag_non_contiguous_weight_cpu_int32_int64_float32 and not test_embedding_bag_non_contiguous_weight_cpu_int64_int32_float32 and not test_embedding_bag_non_contiguous_weight_cpu_int64_int64_float32 and not test_EmbeddingBag_discontiguous and not test_EmbeddingBag_mean and not test_EmbeddingBag_sparse and not test_EmbeddingBag_sum and not test_share_memory" 2>&1 | tee $@.out
```

Comment:
Here we need a variable to represent the set of tests that we want to exclude. I think you can use "\" to separate a string variable into multiple lines. 

Same applies to the excluded tests of `test-autograd` abd `test-autograd-thread-shutdown`.

Reply:
Fixed.